### PR TITLE
refactor : integration popup tracking span & change component tracking span on page detail diverted to span

### PR DESCRIPTION
**Pull request by marsellavaleria19**
## Changes
- format date create span & remove list tracking span on page detail diverted to span
-  change component tracking span on page detail diverted to span
-  integration data & add condition no tracking data on component tracking span
- add locale on function format date utils file

## Evidence
title:  integration popup tracking span & change component tracking span on page detail diverted to span
project: Jabar Super Apps
participants: @marsellavaleria19 @Ibwedagama @rayfajars @yoslie

### CODE REVIEW

**Review by Ibwedagama — APPROVED** [review]

File: components/Aduan/Dialog/TrackingSpanLapor/index.vue [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import CardSpanLapor from '~/components/Aduan/Dialog/TrackingSpanLapor/CardSpanLapor'
+export default {
+  name: 'DialogDetailStatusSpanLapor',
+  components: { CardSpanLapor },
+  props: {
+    showPopup: {
+      type: Boolean,
+      default: true
+    },
+    idApi: {
```
Comment (rayfajars) [inline components/Aduan/Dialog/TrackingSpanLapor/index.vue:41]: is this used?

I saw he sent props from parents, but in the end they weren't used

File: components/Aduan/Detail/index.vue [hunk 2]
```diff
@@ -228,10 +228,10 @@
       </template>
     </basetabgroup>
     <DialogViewImage :list-photo="listPhoto" :show-popup="isShowPopupViewImage" @close="isShowPopupViewImage=false" />
-    <DialogDetailStatusSpanLapor
+    <DialogTrackingSpanLapor
       :show-popup="isShowPopupDetailStatusComplaint"
       :data-dialog="dataDialog"
-      :list-tracking-span-lapor="listTrackingSpanLapor"
+      :id-api="$route.params.id"
```
Comment (rayfajars) [inline components/Aduan/Detail/index.vue:234]: This is what I mean by parent props

**Review by yoslie — APPROVED** [review]

File: components/Aduan/Dialog/TrackingSpanLapor/index.vue (hunk 1 above)
Comment (marsellavaleria19) [reply components/Aduan/Dialog/TrackingSpanLapor/index.vue:41]: Hello @rayfajars .. Yes, I used it for fetching data on dialog tracking span.

Comment (marsellavaleria19) [reply components/Aduan/Dialog/TrackingSpanLapor/index.vue]: Hello @rayfajars , no, it not used.. Thank you for comment. I've fixed it, this is my commit : https://github.com/jabardigitalservice/super-app-cms/pull/116/commits/fe2e17632960979a85d27652d79ef189a9024083

**Review by rayfajars — APPROVED** [review]

**Pull request merged**